agent/engine/docker_image_manager.go

```diff
@@ -214,21 +216,42 @@ func (imageManager *dockerImageManager) removeImageState(imageStateToBeRemoved *
 }
 
 func (imageManager *dockerImageManager) getCandidateImagesForDeletion() []*image.ImageState {
-	if len(imageManager.imageStatesConsideredForDeletion) < 1 {
+	return imageManager.getCandidateImagesForDeletionHelper(imageManager.imageStatesConsideredForDeletion)
+}
+
+func (imageManager *dockerImageManager) getCandidateImagesForDeletionHelper(candidateImagesConsiderForDeletion map[string]*image.ImageState) []*image.ImageState {
```

Comment:
Why using a helper? The only thing that `getCandidateImagesForDeletion` does is to call `getCandidateImagesForDeletionHelper`